examples/webpack2/src/main.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { Button, Container, Header } from 'semantic-ui-react'
```

Comment:
This cannot use tree shaking since it is importing commonjs modules. Use the ES build. 

https://github.com/Semantic-Org/Semantic-UI-React/issues/524#issuecomment-289218645